### PR TITLE
Graded assignment

### PR DESCRIPTION
**Pull request by Cory-Christensen**
Should reflect whether an assignment is graded or ungraded

### CODE REVIEW

File: client/js/scorm/components/scorm/course.jsx [hunk 1]
```diff
@@ -5,12 +5,14 @@ import { connect }              from 'react-redux';
 
 const GradedAssign = (props) => {
   return (
-    <button className="c-icon-btn" style={{display: (props.isGradeActive) ? "inline" : "none"}}>
-      <svg className="c-icon" xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
-          <path d="M0 0h48v48h-48z" fill="none"/>
-          <path className="c-path" d="M28 4h-16c-2.21 0-3.98 1.79-3.98 4l-.02 32c0 2.21 1.77 4 3.98 4h24.02c2.21 0 4-1.79 4-4v-24l-12-12zm4 32h-16v-4h16v4zm0-8h-16v-4h16v4zm-6-10v-11l11 11h-11z"/>
-      </svg>
-    </button>
+    <a href='#'>
```
Comment (Cory-Christensen) [inline client/js/scorm/components/scorm/course.jsx]: Link to assignment

**Comment by dittonjs** [comment]
i think this looks good

**Comment by Cory-Christensen** [comment]
@dittonjs Added one final change if you could review that.

File: client/js/scorm/components/scorm/course.jsx [hunk 2]
```diff
@@ -5,27 +5,27 @@ import { connect }              from 'react-redux';
...
+export class Course extends React.Component {
   constructor(props){
     super(props);
 
     this.state = {
       isGoBtnActive: false,
-      selectVal: "0",
-      isGradeActive: false
+      selectVal: this.formatGraded(props.course),
+      isGradeActive: props.course.is_graded
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/course.jsx]: Should just use the prop?

File: client/js/scorm/components/scorm/course.jsx [hunk 3]
```diff
@@ -5,27 +5,27 @@ import { connect }              from 'react-redux';
...
   }
 
   static propTypes = {
     course: React.PropTypes.object.isRequired
   };
 
-  handleGraded(){
-    const courseId = this.props.course.id;
+  assignmentRedirect(){
+    return 'https://' + this.props.canvasUrl + '/courses/' + this.props.courseId +
```
Comment (BenHSmith13) [inline client/js/scorm/components/scorm/course.jsx]: interpolate this.

**Comment by Cory-Christensen** [comment]
Merged into Nick's branch...will continue there.

**Pull request closed without merging**